microlith/Dockerfile

```diff
@@ -274,7 +275,11 @@ ENV PROMETHEUS_DYNAMIC_DIR=/etc/prometheus/couchbase/
 # Add all useful and required information
 COPY licenses/* /licenses/
 COPY README.md /help.1
-COPY git-commit.txt /etc/cmos-release.txt
+
+# TODO (CMOS-12): this doesn't work in the OSS build (and uses CBMM's version, not CMOS's)
```

Comment:
Yeah this was deliberately detaining the cluster monitor one: I wanted to capture the specific commit we use for that during the build process. We do not need one for CMOS as we'll include it in the build, although it won't hurt to have one.

Reply:
Ack, although in that case I wouldn't call the file `cmos-release`, maybe more like `cluster-monitor-release`. Will add a note to CMOS-12.